src/main/java/io/aiven/kafka/connect/gcs/GcsSinkConfig.java

```diff
@@ -113,17 +118,22 @@ private static void addGcsConfigGroup(final ConfigDef configDef) {
                 "Explicit GCS Endpoint Address, mainly for testing", GROUP_GCS, gcsGroupCounter++, ConfigDef.Width.NONE,
                 GCS_ENDPOINT_CONFIG);
         configDef.define(GCS_CREDENTIALS_PATH_CONFIG, ConfigDef.Type.STRING, null, ConfigDef.Importance.LOW,
-                "The path to a GCP credentials file. "
-                        + "If not provided, the connector will try to detect the credentials automatically. "
-                        + "Cannot be set together with \"" + GCS_CREDENTIALS_JSON_CONFIG + "\"",
+                "The path to a GCP credentials file. " + "Cannot be set together with \"" + GCS_CREDENTIALS_JSON_CONFIG
```

Comment:
```suggestion
                "The path to a GCP credentials file. Cannot be set together with \"" + GCS_CREDENTIALS_JSON_CONFIG
```

Reply:
These must have slipped in with a `spotlessApply` reformat. Resolved all instances of odd string building.